frameworks/g-cloud-10/metadata/copy_services.yml

```diff
@@ -8,9 +8,8 @@ questions_to_copy:
   - APIAutomationToolsOther
   - APIDocumentation
   - APIDocumentationFormats
-  - APIHosting
```

Comment:
The yaml file for APIHosting and APISoftware don't specify an ID, they just have the name API.
Is the question identified by the name if no ID is specified? Is the filename not used at all?

Reply:
Yep, you're right. Was just running and testing it locally but you beat me to it. APIHosting/APISoftware are the keys used.

Reply:
OK, so is the logic that is uses the filename, unless an ID is specified?

Reply:
Yep